components/Input/Input.tsx

```diff
@@ -0,0 +1,79 @@
+import { InputHTMLAttributes } from "react";
+
+interface InputProps
+  extends Pick<
+    InputHTMLAttributes<HTMLInputElement>,
+    "id" | "type" | "value" | "onChange" | "name" | "required" | "className"
+  > {
+  error?: boolean;
+  label: string;
+  helperText: string;
+}
+
+const Input = ({
+  id,
+  error = false,
+  label,
+  required,
+  helperText,
+  className,
+  ...props
+}: InputProps) => (
+  <div className="flex flex-col">
+    <label
+      htmlFor={id}
+      className={`
+        font-medium text-base transition-3 transition-all flex flex-col focus:font-bold
+        ${error ? "text-red" : ""}`}
+    >
+      <span className="h-[35px] block">{`${
+        required ? "*" : ""
+      } ${label}`}</span>
```

Comment:
Hay que evitar en lo posible de agregar `height` y tambien unidades `px`, de dejo una sugerencia que simplifica esto sin mucho (o nada) de CSS:

```suggestion
      <div>
        {required && <span>*</span>}
        {label}
      </div>
```